plugins/Toolbox/resources/qml/ToolboxConfirmUninstallResetDialog.qml

```diff
@@ -19,7 +19,7 @@ UM.Dialog
 
     title: catalog.i18nc("@title:window", "Confirm uninstall ") + toolbox.pluginToUninstall
     width: 450 * screenScaleFactor
-    height: (30 + dialogText.height) * screenScaleFactor
+    height: (100 + dialogText.height) * screenScaleFactor
```

Comment:
Should be ```100 * screenScaleFactor + dialogText.height```
dialogText.height is already DPI-aware.

Reply:
Ahhh thanks ^^